src/main/kotlin/com/codingchallenge/transactionanalyser/TransactionLoader.kt

```diff
@@ -0,0 +1,35 @@
+package com.codingchallenge.transactionanalyser
+
+import com.codingchallenge.transactionanalyser.domain.Transaction
+import com.codingchallenge.transactionanalyser.domain.TransactionBuilder
+import java.io.BufferedReader
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.util.stream.Collectors
+
+class TransactionLoader(private val input: InputStream) {
+    fun loadTransactionsFromCsv(): List<Transaction>{
+        val transactions: List<Transaction>
+        BufferedReader(InputStreamReader(input)).use {
+            transactions = it.lines()
+                .skip(1)
```

Comment:
here you are doing `skip(1)` 
Again come back to my previous comment about cognitive load on a new reader of this code.
I know you skipped 1 because it's the heading row. But think about someone guessing that or trying to see you are doing that without knowing why you had to. 
Could we extract that into a separate function, perhaps, call it `skipHeaderRow()` which would make it more obvious to the reader? Maybe not that easy on the InputStreamReader but something to just have a think about.

Reply:
Yeah, should be addressed by using an external csv library.